cmd/kogito/command/shared/install_operator.go

```diff
@@ -113,14 +129,18 @@ func installOperatorWithYamlFiles(image string, namespace string, cli *client.Cl
 			return err
 		}
 	}
-
-	if err := decodeAndCreateKubeObject(box, fileServiceAccountYaml, &v1.ServiceAccount{}, namespace, cli, nil); err != nil {
+	if err := decodeAndCreateKubeObject(box, fileNamespaceYaml, &v1.Namespace{}, namespace, cli, nil); err != nil {
 		return err
 	}
-	if err := decodeAndCreateKubeObject(box, fileRoleYaml, &rbac.Role{}, namespace, cli, nil); err != nil {
+	if err := decodeAndCreateKubeObject(box, fileClusterRoleYaml, &rbac.ClusterRole{}, namespace, cli, nil); err != nil {
 		return err
 	}
-	if err := decodeAndCreateKubeObject(box, fileRoleBindingYaml, &rbac.RoleBinding{}, namespace, cli, nil); err != nil {
+	if err := decodeAndCreateKubeObject(box, fileServiceAccountYaml, &v1.ServiceAccount{}, namespace, cli, nil); err != nil {
+		return err
+	}
+	if err := decodeAndCreateKubeObject(box, fileClusterRoleBindingYaml, &rbac.ClusterRoleBinding{}, namespace, cli, func(object interface{}) {
```

Comment:
Minor - In case the ClusterRoleBinding already exists then it would be fine just to add the new service account there, allowing namespaced installation of several independent Kogito operators.

Reply:
This is a edge case in which user wants to install the operator in a specific namespace without the OLM, Since currently we don't have the idempotent approach, which should be implemented in [this](https://issues.redhat.com/browse/KOGITO-3880). So our option would be to delete the object and recreate with the new references. IMO we should not handle this right now, and can be worked on after we have the idempotent approach. Also I think we should also encourage user to use the cluster-wide option instead of the `namespaced` one Specially in the `yamlInstallation`.

Reply:
ok